src/ui/components/shared/WorkspaceSettingsModal/WorkspaceSubscription.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import hooks from "ui/hooks";
+
+import APIKeys from "../APIKeys";
+
+export default function WorkspaceSubscription({ workspaceId }: { workspaceId: string }) {
+  const {
+    addWorkspaceApiKey,
+    loading: addLoading,
+    error: addError,
+  } = hooks.useAddWorkspaceApiKey();
+  const { data } = hooks.useGetWorkspaceSubscription(workspaceId);
+  const { activateWorkspaceSubscription, loading: activateLoading } =
+    hooks.useActivateWorkspaceSubscription();
+
+  return (
+    <section>
+      {data?.node.subscription ? (
+        "Current sub: " + data?.node.subscription.plan.key
+      ) : (
+        <>
+          <p>Get started now for $20/Debugger/month!</p>
+          <button
+            disabled={activateLoading}
+            onClick={() => {
+              if (activateLoading) return;
+
+              activateWorkspaceSubscription({
+                variables: {
+                  workspaceId,
+                  seatCount: 1,
+                  subscriptionKey: "team-1",
+                },
+              });
+            }}
+          >
+            Start with 1 Seat
+          </button>
```

Comment:
This isn't relevant right now. In the future, we'll need something like this to help transition beta teams onto paid plans but it's cruft that we don't need right now.